mdx-models/src/main/java/com/mx/path/model/mdx/model/challenges/Question.java

```diff
@@ -2,213 +2,35 @@
 
 import java.util.List;
 
-import com.google.gson.annotations.SerializedName;
+import lombok.Getter;
+import lombok.Setter;
+
 import com.mx.path.model.mdx.model.MdxBase;
 import com.mx.path.model.mdx.model.MdxNested;
 
 @MdxNested
+@Getter
+@Setter
```

Comment:
I've been trying to standardize the headers in MDX models. Can you change this to the following?

```java
@MdxNested
@Data
@EqualsAndHashCode(callSuper = true)
```
The `@Data` annotation includes `@Getter` and `@Setter`.